outputs.tf

```diff
@@ -24,7 +24,7 @@ output "attributes" {
 }
 
 output "tags" {
-  value       = "${local.tags}"
+  value       = "${local.tags[var.user_tags_only ? "user" : "all"]}"
```

Comment:
Also, you've mentioned tags as normalized . In fact there's no normalization... keys are untouched and basically everything is outputted exactly as it was passed via variable...